Add _afterPaintQueued and _fallback to mangle.json

38elements opened the pull request.

38elements commented on hooks/mangle.json:
```diff
@@ -6,6 +6,7 @@
   "props": {
     "cname": 6,
     "props": {
+      "$_afterPaintQueued": "__a",
```
https://github.com/preactjs/preact/blob/0aeca9ac0bc089d5df4549b1ef54a15de9f135a8/hooks/src/index.js#L220

38elements commented on compat/mangle.json:
```diff
@@ -20,6 +20,7 @@
       "$_root": "__p",
       "$_commit": "__c",
       "$_suspensions": "__u",
+      "$_fallback": "__",
```
https://github.com/preactjs/preact/blob/b051204ca2dbcd409288a6a74d1ec8fbeae94fd9/compat/src/suspense.js#L46

38elements replied:
@JoviDeCroock 
Thank you for review.
I am sorry.
I changed this line for #2044.

andrewiggins commented on compat/mangle.json:
```diff
@@ -20,11 +20,12 @@
   }
-}
\ No newline at end of file
+}
```
Nit: This file shouldn't end in a new line. One of the build scripts writes this file after every build and writes it without a final new line. Build all again and you should see a change that removes this final new line (at least it does on my machine 🤷‍♂ ).

38elements replied:
@andrewiggins
Thank you for review. 
I removed the last empty line.

38elements commented:
@JoviDeCroock @andrewiggins 
I am sorry.
I removed the end line by the editor of Github.
But, the end line was not removed. 
I removed it at #2053.

andrewiggins commented:
No worries! Thanks for the PRs!
